.github/workflows/terraform.yml

```
@@ -67,8 +67,8 @@ jobs:
 
       - name: Test with Checkov
         id: checkov
-        uses: bridgecrewio/checkov-action@master
+        uses: bridgecrew/checkov-action@master
         with:
           directory: ./
           framework: terraform
-          skip_check: CKV_TF_1
+          skip_check: CKV_TF_1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)